src/cli.mjs

```diff
@@ -64,23 +70,28 @@ program.parse();
 const files = program.args;
 console.log("Processing files: ", files);
 
-const { transpile, outputDir, transformsFile, checkCompat } = program.opts();
+const { transpile, outputDir, transformsFile, transpileIgnore } =
+  program.opts();
 const transforms = await buildTransformsChain(transformsFile);
-const bundle = await doBundleAndTranspile(files, outputDir, { transforms });
+const bundle = await doBundleAndTranspile(files, outputDir, {
+  transforms,
+  transpileIgnore: [transpileIgnore].flat(),
```

Comment:
**Annotation:**
In the CLI, we allow the passing of a single glob. However, if the API is `import`ed and invoked in a script, it takes an array.  Therefore for the CLI use case, we convert the glob to an array to respect the interface of the core logic module.